modules/mvt/src/lib/binary-vector-tile/features-to-binary.ts

```diff
@@ -513,6 +519,14 @@ function keepStringProperties(
   return props;
 }
 
-function isNumeric(x: unknown) {
-  return Number.isFinite(x);
+function deduceArrayType(
```

Comment:
Why does this code need to be repeated here? Is there any way we can unify things a bit?